cmd/pool/auction.go

```diff
@@ -138,18 +151,27 @@ func batchSnapshot(ctx *cli.Context) error {
 		return fmt.Errorf("unable to decode batch ID: %v", err)
 	}
 
-	ctxb := context.Background()
-	batchSnapshot, err := client.BatchSnapshot(
-		ctxb,
-		&poolrpc.BatchSnapshotRequest{
-			BatchId: batchID,
-		},
-	)
+	if ctx.Uint64("num_batches") > 1 {
+		resp, err = client.BatchSnapshots(
+			ctxb, &poolrpc.BatchSnapshotsRequest{
+				StartBatchId: batchID,
+				NumBatchesBack: uint32(ctx.Uint64(
+					"num_batches",
+				)),
+			},
+		)
+	} else {
+		resp, err = client.BatchSnapshot(
+			ctxb, &poolrpc.BatchSnapshotRequest{
+				BatchId: batchID,
```

Comment:
Took a look at the server and it requires a positive `num_batches_back` so we need to specify it here in case it's been set to 1.

Reply:
We call the singular RPC here which doesn't have that parameter. But perhaps we can just deprecate that one and only use the new, multi batch RPC.

Reply:
Ah didn't realize that. I thought the plan was to deprecate that one eventually anyway?